azure-mgmt-compute/src/main/java/com/microsoft/azure/management/compute/VirtualMachine.java

```diff
@@ -1850,6 +1923,57 @@ Update withExistingDataDisk(Disk disk,
             @Beta(Beta.SinceVersion.V1_2_0)
             Update withoutBootDiagnostics();
         }
+
```

Comment:
is it not possible to turn MSI off in an update?  (I don't see `.withoutManagedServiceIdentity()`)